src/goSuggest.ts

```diff
@@ -53,7 +53,15 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				let autocompleteUnimportedPackages = config['autocompleteUnimportedPackages'] === true && !lineText.match(/^(\s)*(import|package)(\s)+/);
 
 				if (lineText.match(/^\s*\/\//)) {
-					return resolve([]);
+					let nextLine = document.lineAt(position.line + 1).text;
+					let memberName = nextLine.replace(/\s+/g, ' ').trim().split(' ');
+					if (!nextLine.startsWith('\t')) {
+						// check for exported members declared in block
+						memberName.shift();
+					}
+					if (!memberName[0].match(/^[A-Z]/)) {
```

Comment:
Say we found an exported member. In that case we should return a single item (the member name) in the suggestion list. Currently `gocode` gets called and it can return multiple results.

Reply:
In this case gocode should not be invoked right?

Reply:
Yes, we shouldn't invoke `gocode`. Instead we should return a single completion item with the member name right here

Reply:
Cool, will do.